### ISSUE TITLE
Problem with wet clothing after activate settings "Frozen in place"

### ISSUE THREAD

**Issue by pat225** [itsjustcurtis/MenyooSP#224]
Hello. I find problem, that, when I spawn ped and I set for ped "Frozen in place", and after I put him to water, ped can not be wet. But when ped is "Frozen in place" and is rain/storm, ped CAN be wet. So, in my opinion, this is problem, that frozen ped placed in water, can not be wet, but ped frozen in place when is rain/storm, can be wet.

**Comment by itsjustcurtis** [comment]
Is this referring to a visual glitch that the peds clothing is not visibly wet?

**Comment by pat225** [comment]
Yes, when it rains, clothes look wet on frozen ped, but when i put frozen ped to water, clothes are dry. Don´t looks wet.
![Dry clothes](https://github.com/user-attachments/assets/e85e0956-7155-4ba8-a145-f9b9a05179cb)
![wet clothes](https://github.com/user-attachments/assets/b2c23498-2c89-44bf-b4ec-a3fdf4ad7703)

On first picture I froze ped and place to water - is dry. On second picture is ped still frozen and I set rain. Is wet.

**Comment by TayMcKenzieNZ** [comment]
Can you replicate this with Freemode peds and Michael?

**Comment by pat225** [comment]
Yes, I try it and send answer.

**Comment by itsjustcurtis** [comment]
![image](https://github.com/user-attachments/assets/03467b95-da86-4397-b4e6-818482867d7b)

It looks like these natives can be used to manipulate wetness. Whilst having the wetness set by the game based on its water position, if my suspicions are correct that this is a collision issue caused by the frozen in place marker, setting it through these natives may be the better option. It may also be prudent to include these as additional controls in the ped menu itself.
